packages/governance/src/types.js

```diff
@@ -0,0 +1,34 @@
+// @ts-check
+
+/**
+ * @typedef {'nat' | 'bigint' | 'string' | 'ratio' | 'amount' | 'brand' } ParamType
+ */
+
+/**
+ * @typedef  {Object} ParamManager
+ * @property {(name: string, value: any) => void} update
```

Comment:
If we restrict types, we could tighten up this `any` to be the types we support. This would help developers a lot because they could find bugs statically before deploying (or testing locally, I suppose).

```suggestion
 * @property {(name: string, value: ParamValue) => void} update
```

Reply:
If I thought we had gotten everything that devs might want to put in a contract, I would be in favor of this, but I think devs are going to need an escape hatch. Having the explicit `ParamType.ANY`  lets them clearly escape our limits. We should try to be welcoming to new suggestions, but there's clearly going to be a release cycle before we can get them support for new types.